data/crac-creation/crac-creator-cse/src/main/java/com/powsybl/openrao/data/craccreation/creator/cse/outage/TOutageAdder.java

```diff
@@ -55,7 +55,6 @@ private void importContingency(TOutage tOutage) {
         tOutage.getBranch().forEach(tBranch -> handleTBranch(tBranch, contingencyAdder, contingencyElementHelpers, outageId));
         if (atLeastOneBranchIsMissing(contingencyElementHelpers)) {
             addNotAddedOutageCreationContext(tOutage, contingencyElementHelpers);
-            crac.removeContingency(outageId);
```

Comment:
This line is removed, where is the equivalent with core Contingency?

Reply:
This line is no longer needed after a bug fix. The fix remove an incorrect add() in handleTBranch so contingency is only added in the "else" below, so no need to remove it in the "if" (in addition to that there is also no more contingency's elements added in the Crac before add() so there is no need to remove them neither).